### PR TITLE
Add missing information from closed and recently opened PRs.

### LINKED ISSUE #49: Update CHANGELOG with missed information
## Depends on tasks:
none

## Description:
Update the changelog file with missed information.

### User requirements:
- Add the changes to[ CHANGELOG ](https://github.com/ios-course/ironfoudation-team-project/blob/main/CHANGELOG.md)file from [missed PRs](https://github.com/ios-course/ironfoudation-team-project/pulls?q=is%3Apr).

### Dev requirements and restrictions:
- check carefully all closed PRs and update the CHANGELOG file simultaneously.

## UI Mocks
not required

## Other useful information
none

### PR DESCRIPTION
**Pull request by NadzeyaShpakouskaya**
## Summary
This PR is a part of the work to resolve #49.
Add missing information to `CHANGELOG.md` from closed and recently opened PRs.

## Changes
- Update `CHANGELOG.md`  with the missing information.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,16 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 
 ### Added 
 - Add the initial Xcode project.
-- Add the Swiftlint script to the build phase of the Woof target.
-- Add the Swiftformat script to the build phase of the Woof target.
+- Add the `Swiftlint` script to the build phase of the Woof target.
+- Add the `Swiftformat` script to the build phase of the Woof target.
 - Update the deployment target to 15.0+. 
+- Add `Owner` model to the model layer.
```
Comment (julyakmita) [inline CHANGELOG.md:14]: Missed:
- Create `project structure` according to MVVM architecture pattern.
or
-Add  'code's organization` by layers according to MVVM.

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,16 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
...
+- Add `Sitter` model to the model layer.
+- Add `AppStyle` folder with extensions files described App Style Guideline used in the application.
+- Add `MainTabView` for the owner with stub view for tabs.
+- Add `SitterCardView` as part of `SitterListView`.
+- Add `SitterListView` to the Sitter's tab in `MainTabView`.
+- Add `OwnerCardView` as part of `OwnerInformationView`.
+- Add `OwnerInformationView` to the Profile's tab in `MainTabView`.
+- Add `DetailedSitterView` to present the detailed information about the selected sitter from `SitterListView`.
+- Add templates for PR and issues.
```
Comment (julyakmita) [inline CHANGELOG.md:23]: Shall we mention this here?

**Review by julyakmita — CHANGES_REQUESTED** [review]
Well done! One question:

File: CHANGELOG.md [hunk 3]
```diff
@@ -8,6 +8,16 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 
 ### Added 
 - Add the initial Xcode project.
-- Add the Swiftlint script to the build phase of the Woof target.
-- Add the Swiftformat script to the build phase of the Woof target.
+- Add the `Swiftlint` script to the build phase of the Woof target.
+- Add the `Swiftformat` script to the build phase of the Woof target.
 - Update the deployment target to 15.0+. 
+- Add `Owner` model to the model layer.
+- Add `Sitter` model to the model layer.
```
Comment (dersim-davaod) [inline CHANGELOG.md:15]: nit:

It's a good practice to put link to the PR:

```suggestion
- Add `Sitter` model to the model layer - [#xx](link-here).
```

That helps to find corresponding PRs later easily.

**Comment by dersim-davaod** [comment]
LGTM

I suggest to consider one more thing.

**Review by dersim-davaod — APPROVED** [review]
LGTM

I suggest to consider one more thing.

File: CHANGELOG.md (hunk 1 above)
Comment (NadzeyaShpakouskaya) [reply CHANGELOG.md]: Added missing PRs, thanks.

File: CHANGELOG.md (hunk 3 above)
Comment (NadzeyaShpakouskaya) [reply CHANGELOG.md]: Added the links

**Review by julyakmita — APPROVED** [review]
LGTM

**Pull request merged**